ammo-fe/src/network/WS.ts

```diff
@@ -1,24 +1,61 @@
-import { io } from 'socket.io-client';
-import MockedSocket from './MockedSocket';
-
```

Comment:
Why is there that much changes in this file ? The merge must have fucked up, it seems like a rollback